packages/components/organisms/f-checkout/src/services/AccountServiceApi.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
```

Comment:
Following on from [Sacha's comment](https://github.com/justeat/fozzie-components/pull/1112#pullrequestreview-706006432) we should be using consistent casing in the module filenames. Camel-case is standard in JS so I'd say go with that — Pascal-case tends to be used when a module exports a class or function that should be instantiated with the `new` keyword.

Reply:
Makes sense, thanks both - I'll update the file names so they all use camel-case 👍